src/dl_node.cpp

```diff
@@ -71,12 +71,14 @@ Status DLNode::fetchResults(TensorWithSourceMap& outputs, ov::InferRequest& infe
         SPDLOG_LOGGER_ERROR(dag_executor_logger, "Node: {} session: {} exception occured during infer request wait: {}", getName(), sessionKey, e.what());
         return StatusCode::INTERNAL_ERROR;
     }
+    double ovInferTime = this->getNodeSession(sessionKey).getTimer().elapsed<std::chrono::microseconds>("inference");
```

Comment:
If we would build OVMS without DEBUG logs in previous version this would not be executed?